content/en/role-based-access-control.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Role-Based Access Control (RBAC)
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+Role-Based Access Control (RBAC) is a method to regulate system and resource access based on individual users' roles within an organization.
+Each job function within an organization has a different role with specific permissions. 
+For instance, a marketing team member may have permission to view the deals sales has in the pipeline (*read* permission) but won't be allowed to edit them (*write* permission). 
+Sales and marketing have different permissions based on their roles, hence role-based access control. 
+
+## Problem it addresses
+
+Without RBAC, it is easy to lose track of who has access to what. Malicious actors are always in search of vulnerabilities to exploit and a mismanaged account with too many privileges can put an entire system at risk. 
+
+
+## How it helps
+
+It simplifies access management and enables more granular control of user actions in software applications.
+The allocation of features to the right people prevents overprivileged accounts and unauthorized access.
```

Comment:
```suggestion
This granular permission allocation allows various team members with different roles to work on the same system without risking...
```
What does "overprivileged accounts and unauthorized access" mean concretely?

Reply:
I would say that both statements are somehow correct. Overpriviliged would mean that users or groups have more permissions than they need, while unauthorized ones would have fewer permissions than they need?!

Reply:
As Noah said, overprivileged means that users or groups have more permissions than they need. The same applies to unauthorized access; users or groups have access to systems they shouldn't have.